### DISCUSSION TITLE
Convert Docs to vitepress

### DISCUSSION THREAD

**Discussion by gldtn** [nikitabobko/AeroSpace#904] [feature-ideas]
@nikitabobko would you consider a conversion of [Aerospace docs](https://nikitabobko.github.io/AeroSpace/guide) to [Vitepress](https://vitepress.dev/guide/what-is-vitepress)? I started converting it, but before I go all out I wanted to check if it's something you would consider it.

**Benefits**:

- Beautiful light/dark mode
- Easy to write(Markdown)
- Searchable docs

If it's something that interest you I can finish the entire conversion for you.

Thanks!

**Comment by nikitabobko** [comment]
Thank you, but it's not something I'm interested in. I'm very happy with asciidoc and the way the site is built right now.
